config/http.conf

```
@@ -44,9 +49,19 @@ http {
     ''      $request_id;
   }
 
+  # See https://www.nginx.com/blog/websocket-nginx
+  map $http_upgrade $connection_upgrade {
+    default upgrade;
+    ''      '';
+  }
+
   proxy_http_version  1.1;
-  proxy_set_header    Connection          "";
   proxy_set_header    Host                $host;
+  proxy_set_header    Connection          $connection_upgrade;;
```

If you agree to have to `$proxy_connection` this should change here also. Also note that there are two `;` at the end:

```suggestion
  proxy_set_header    Connection          $proxy_connection;
```